app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
```

Comment:
optional:
you should make the `:update_post_counter` method private and you should use the rails callback `after-save` to call the method as mentioned above every time a new post is saved to the database 👍🏾